gitpod-io/gitpod
# Add migration for new usage table

## Description

Define the schema and add the migration for a new `d_b_workspace_instance_usage` table to store workspace usage data.

## Related Issue(s)
Part of https://github.com/gitpod-io/gitpod/issues/10323

## How to test

Connect to the database in the preview environment for this branch and see the new table in the database:

<img width="691" alt="image" src="https://user-images.githubusercontent.com/8225907/178475324-2636441d-59aa-4708-8eb0-fa0f5d2353ed.png">

## Release Notes

```release-note
NONE
```

## Documentation

## Werft options:
